tests/schema-validation.js

```diff
@@ -6,6 +6,7 @@ function createGenerator() {
     try {
         return schemaGenerator.createGenerator({
             path: './schema/config.ts',
+            tsconfig: './tsconfig.json',
```

Comment:
This fixed the mentioned issue, but I think we should discuss if it's safe. And/or if we should create a separate tsconfig.